parquet-arrow/src/main/java/org/apache/parquet/arrow/schema/SchemaConverter.java

```diff
@@ -86,10 +86,18 @@
  */
 public class SchemaConverter {
 
+  // Indicates if Int96 should be converted to timestamp arrow type.
+  private final boolean convertInt96ToArrowTimestamp;
+
   /**
    * For when we'll need this to be configurable
    */
   public SchemaConverter() {
+    this(false);
+  }
+
+  public SchemaConverter(final boolean convertInt96ToArrowTimestamp) {
+    this.convertInt96ToArrowTimestamp = convertInt96ToArrowTimestamp;
```

Comment:
Will it be possible that in the future we need to convert int96 to some other type in arrow (or for any other type convertion)? If that's something likely to happen, perhaps it would be better to pass the parameter in a configuration object (such as `org.apache.hadoop.conf.Configuration`)

Reply:
We need to pass this later on as an option from the outside. The current routines don't know anything about the file creator. @yongyanw can you add this as a TODO comment and open a JIRA for it? I don't expect that we can solve this in this PR.

Reply:
PARQUET-1511 was created and TODO comment was added.